### PR TITLE
[WIP] Move shinyngs modules across to nftest

### PR DESCRIPTION
**Pull request by pinin4fjords**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/shinyngs/app/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    name "Test Process SHINYNGS_APP"
+    script "../main.nf"
+    process "SHINYNGS_APP"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "shinyngs"
+    tag "shinyngs/app"
+
+    test("multi - tsv") {
```
Comment (koenbossers) [inline modules/nf-core/shinyngs/app/tests/main.nf.test:12]: From [the migration guide to nf-test](https://nf-co.re/docs/contributing/modules#migrating-from-pytest-to-nf-test): provide a test name preferably indicating the test-data and file-format used. Example: test("homo_sapiens - [bam, bai, bed] - fasta - fai")

File: modules/nf-core/shinyngs/app/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+    test("single - tsv") {
```
Comment (koenbossers) [inline modules/nf-core/shinyngs/app/tests/main.nf.test:60]: See previous remark on naming conventions

**Review by koenbossers — CHANGES_REQUESTED** [review]
Looks good to me! Only a small note on test naming conventions.

**Comment by SPPearce** [comment]
Bumping to see if we can get this merged in

**Comment by pinin4fjords** [comment]
Apologies, I think this is an orphan, I must have forgotten about it and did the work again elsewhere.

**Comment by SPPearce** [comment]
No worries, happy to clean up another PR ;)

**Pull request closed without merging**